engine/schema/src/main/java/com/cloud/upgrade/SystemVmTemplateRegistration.java

```diff
@@ -888,4 +901,17 @@ public void doInTransactionWithoutResult(final TransactionStatus status) {
             }
         });
     }
+
+    public String getNfsVersion(long storeId) {
+        final String configKey = "secstorage.nfs.version";
```

Comment:
`CapacityManager.ImageStoreNFSVersion.key()` ?

Reply:
@weizhouapache it doesn't seem to be possible to use ConfigKey in `SystemVmTemplateRegistration` class as it is part of `engine-schema` module. Other configs are accessed the same way. I tried but it creates a cyclical dependency

Reply:
ok, then it is fine to use  "secstorage.nfs.version"
thanks @shwstppr for the attempt